testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/AbstractTestHiveViews.java

```diff
@@ -443,6 +443,8 @@ public void testCurrentUser()
     }
 
     @Test(groups = HIVE_VIEWS)
+    // Test is currently flaky on CDH5 environment
+    @Flaky(issue = "https://github.com/trinodb/trino/issues/9074", match = "Error while processing statement: FAILED: Execution Error, return code 2 from org.apache.hadoop.hive.ql.exec.mr.MapRedTask")
```

Comment:
See also the two methods mentioned in https://github.com/trinodb/trino/issues/9089 which fail with same cause.

Reply:
Indeed. @Praveen2112  do you want o address #9089 ?

Reply:
Yeah sure !!